source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -299,28 +295,24 @@ package:
        and/or deserialize a property.
 
    * - ``BsonProperty``
-     - Specifies a custom document field name when converting the POJO
-       field to BSON. You can include a discriminator to serialize POJOs
-       nested within the field.
-
-       .. important::
+     - | Specifies a custom document field name when converting the POJO
+         field to BSON. You can include a discriminator to serialize POJOs
+         nested within the field. 
+       | 
+       | When applying ``@BsonProperty`` to a private field, 
+         **you must also add getter and setter methods for that field to serialize and 
+         customize the field name**.
```

Comment:
```suggestion
       | When applying ``@BsonProperty`` to a private field, 
         **you must also add getter and setter methods for that field to serialize and 
         customize the field name**.
```
S: The bolding looks a little too bold for this small section. The Style Guide suggested using the gold Important directive to replace the Important note directive: https://www.mongodb.com/docs/meta/style-guide/nested-components/#important-or-warning.

Reply:
These are being removed as part of this nested component work where we want annotations taken out of tables.

Reply:
Is using the gold Important directive rather than the bolding not necessary here then? Mean this gold directive: https://www.mongodb.com/docs/meta/style-guide/nested-components/#important-or-warning:~:text=IMPORTANT%3A%20Data%20transfer%20as%20part%20of%20the%20backup%20and%20restore%20process%20is%20charged%20separately.

Reply:
Ok, LGTM!